tests/yast2_ui/yast2_snapper.pm

```diff
@@ -67,12 +67,13 @@ sub clean_and_quit() {
     # C'l'ose  the snapper module
     send_key "alt-l";
     # Wait until xterm is focussed, delete the directory and close xterm
-    wait_idle;
+    wait_idle 3;
```

Comment:
yeah, it's some kind of magic. But honestly I don't remember why I changed it :)